pkg/kritis/metadata/metadata.go

```diff
@@ -59,13 +67,36 @@ type Vulnerability struct {
 	CVE             string
 }
 
+// RawAttestation is an internal wrapper for PgpAttestations and
+// GenericAttestations. It contains signatures, which are verified by
+// public keys held by the AttestationAuthority.
+type RawAttestation struct {
+	AttestationType    AttestationType
```

Comment:
I don't think it necessarily does. Qifan and I are thinking that it may be better to use a Signature interface, which PGPAttestation and GenericAttestation will both implement.

Reply:
Fine with me, just keep in mind that then you may end up having to immediately define the public key interface as well.  (You probably have to pass the key to VerifySignature().)

Reply:
I would propose the following:
```
type RawAttestation struct {
SignatureType string
Signatures []RawSignature
Plaintext String
}

type RawSignature struct {
   PublicKeyId string
   Signature string
}
PgpAttestation PGPAttestation
GenericAttestation GenericAttestation
}
```
with reasons discussed in the doc. (maybe with minor tweaks on exported identifier and naming)

Reply:
We discussed offline, and decided to move to this design.